### PR TITLE
fix: add missed template for serviceaccount and add basic RBAC resources

### PR DESCRIPTION
**Pull request by SweetOps**
#### What this PR does / why we need it:
* add missed template for serviceaccount
* add basic RBAC resources
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/mariadb/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: mariadb
-version: 5.5.3
+version: 5.5.4
```
Comment (juan131) [inline stable/mariadb/Chart.yaml]: Please increase the minor version instead.

File: stable/mariadb/templates/role.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{{- if and .Values.serviceAccount.create  .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ template "master.fullname" . }}
+  labels:
+    app: "{{ template "mariadb.name" . }}"
+    chart: {{ template "mariadb.chart" . }}
```
Comment (juan131) [inline stable/mariadb/templates/role.yaml]: For consistency, I'd also quote this label the same way you did with "app" label. It applies to the rest of manifests.

**Review by juan131 — COMMENTED** [review]
Hi @SweetOps 

Could you please provide more context about the scenario where this chart doesn't work if RBAC is enabled? I guess it's an issue with slaves trying to connect with the master nodes but I just want to understand the reason why it's necessary to add permissions to get the endpoints.

**Comment by juan131** [comment]
Hi @SweetOps 

Please revert from master to fix the conflicts and sign your last commits.

**Comment by SweetOps** [comment]
Hi @juan131 , this's just good behaviour when you create `serviceaccount` you should assign some `role` that's will prevent `mind games`  e.g. `What can I do with that serviceaccount?`, `What the potential attack vector someone can use?` and etc. With permissions get the endpoints the attack vector will be very very limited. The role should be `deny everything` but `Permissions are purely additive (there are no “deny” rules)` https://kubernetes.io/docs/reference/access-authn-authz/rbac/

**Review by juan131 — APPROVED** [review]
Thanks for the great explanation @SweetOps !!! Approving your changes.

**Pull request merged**